### ISSUE TITLE
Screen reader does not announce 'Setup progress' status message information which appear after activating 'Install' button in Windows.

### ISSUE THREAD

**Issue by msftedad** [wixtoolset/issues#9109]
### WiX Version

5.0.0

### .NET or MSBuild or Visual Studio Version

.NET 8.0.314

### HeatWave Version

1.0.6

### Repro Steps

**User Impact:**
Screen reader users will be impacted when screen reader does not announce any received message information after a control has been activated. Users will not know whether the control has been activated or not.

**Observation:**
This issue is observed with all three screen readers.

**Pre-Requisite:**
1. Turn on Screen Readers:
Narrator: Win + Shift + Enter
NVDA: Ctrl + Alt + N
JAWS: Ctrl + Alt + J

2. Verbosity:
Narrator: Default
JAWS: Beginner, Highest
NVDA: Default

**Repro Steps:**

1. Launch the "NPS Extension for Azure MFA" setup file.
2. "MICROSOFT SOFTWARE LICENSE TERMS" popup window will get open here.
3. Press tab key to move to "I agree to the terms and conditions check box" and activate it using enter key.
4. Press tab key to move to "Install" button and activate using 'Enter' key.
5. Activate Yes button on the "Do you want to make changes to your device pop-up window"
6. Verify whether Screen reader is announcing 'Setup progress' status message information which appear after activating 'Install' button or not.

### Repro Repo

https://www.microsoft.com/en-us/download/details.aspx?id=54688

### Actual Result

Screen reader does not announce 'Setup progress' status message information which appear after activating 'Install' button in Windows.
Narrator remains silent.

**Refer Attachment:**
1. Screen reader does not announce 'Setup progress' status message information which appear after activating 'Install' button in Windows.png
2. Screen reader does not announce 'Setup progress' status message information which appear after activating 'Install' button in Windows.mp4

<img width="788" alt="Image" src="https://github.com/user-attachments/assets/754d9a37-dead-4e8f-978f-8719a8a10df1" />
https://github.com/user-attachments/assets/74585eec-0fda-430e-81c7-a6e08094cac0

### Expected Result

Screen reader should announce 'Setup progress' status message information which appear after activating 'Install' button in Windows.
Narrator should announce as " Setup progress 20% completed".

### Acknowledgements

- [x] I acknowledge that this is a fully completed bug report. It is not a question or attempt to get help debugging my issue (because those should be [sent to Discussions](https://github.com/orgs/wixtoolset/discussions/new?category=questions)).
- [x] I am paying the [Open Source Maintenance Fee](https://opensourcemaintenancefee.org/) by [sponsoring the `wixtoolset` project](https://github.com/sponsors/wixtoolset) because I support the maintainers.
